README.md

```diff
@@ -12,36 +29,30 @@ We love translations of our guides to new languages. That helps the Storybook co
 
 Traditional Chinese translation is converted from Simplified Chinese using [OpenCC](https://github.com/BYVoid/OpenCC). Please help us correct any idiomatic errors.
 
-## Who made LearnStorybook.com?
+### Running the project locally
 
-The text, code, and production were contributed by [Chromatic](https://www.chromatic.com/?utm_source=storybook_website&utm_medium=link&utm_campaign=storybook). The tutorial was inspired by Chromatic’s popular [GraphQL + React tutorial series](https://www.chromatic.com/blog/graphql-react-tutorial-part-1-6)
+#### Storybook instructions
 
-## Develop
+The Storybook for Storybook contains every UI component. The UI is built following [Component-Driven Development](https://blog.hichroma.com/component-driven-development-ce1109d56c8e), a process that builds UIs from the “bottom up” starting with components and ending with screens. That means contributors should compose UIs in Storybook _before_ integration with the Gatsby app.
 
-Install packages `yarn`
+1. Run `yarn install` to install dependencies
+2. Run `yarn storybook` to start Storybook on `localhost:6006`
 
-This project relies on some content from the main Storybook repo. To fetch it, run:
+#### Gatsby instructions
 
-```bash
-yarn extract-sb-docs-metadata
-```
+Gatsby is used for static site generation. To run the project locally, you'll need to:
 
-Start development server with `yarn dev`
+1. Run `yarn install` to install dependencies
+2. Run `yarn extract-sb-docs-metadata` to fetch content from the main Storybook repo
+3. Set up a `.env.development` file with the following environment variables:
 
-## Contributors
+```plaintext
+SKIP_DX_DATA=true
+```
```

Comment:
If this is always desired, would it make sense to commit `.env.development` with this set?